tools/http_cache/main.go

```diff
@@ -16,7 +16,7 @@ var opts = struct {
 	Usage     string
 	Verbosity cli.Verbosity `short:"v" long:"verbosity" default:"warning" description:"Verbosity of output (higher number = more output)"`
 	CacheDir  string        `short:"d" long:"dir" default:"" description:"The directory to store cached artifacts in."`
-	Port      int           `short:"p" long:"port" description:"The port to run the server on"`
+	Port      int           `short:"p" long:"port" description:"The port to run the server on" default:"8080"`
```

Comment:
not related to this change?

Reply:
Ah sorry, I was doing some work on testing the cache. Probably nice to have a default though...